dansantander/re-former
# #Implement-project-Re-former

**What Does This PR do?**
This PR is for the TSE review of our project implementation.

**NOTE**
- ensure the minimum length of test values are supplied

**How can This be Manually Tested?**
- After installing the project
- Run `bundle install`, then => run  `rails db:migrate`
- Start the server of the project by running `rails server`
- Head over to your browser on localhost:3000 or whatever port the project is said to be running on your terminal
**Testing creation of a new User**
- navigate to the route localhost:3000/users/new and you'll be greeted with a basic HTML/.erb form UI 
- input desired values in the available  input fields and hit the `create` button
- To check if the transaction was successful, head over to your terminal(preferably open a new terminal and navigate to the project directory)
- enter the command  `rails console`
- then run  `User.all`
- if the values you passed on the front end are seen in the users record, then voila! it worked.

**Testing Update of the record**
- Here to test this transaction, you must know the `id` of which particular user you want to edit
- Head over to localhost:3000/users/user_id/edit 
- Here you'll be greeted with the edit form
-  you can pass in entirely new or all of any of the values of the user you wish to edit
- Hit the submit button
- To check if the transaction was successful, follow the steps above and access the rails console once again, if the update values were effected on the old record, the transaction was successful!
